index.html

```diff
@@ -134,6 +134,25 @@ <h2>Connect</h2>
         <a href="https://www.github.com/hzeng33" class="fa fa-github"></a>
       </div>
     </section>
+
+    <section>
+      <h2>Leave a Message</h2>
+      <form id="leave_message">
```

Comment:
One small suggestion: it might be better to use the `name` attribute instead of `id` to follow the requirements of the assignment (although your current implementation is correct!).